### PR TITLE
feat: make background image responsive

### PR DESCRIPTION
**Pull request by Ismailtlem**
Hi,

This PR is for making the background image responsive.
The animation is present for screens with width > 750 px
For screens with width < 750 px, I keep only the background image
For screens with width < 500 px, a blue background is displayed

This PR is for the issue #105

### CODE REVIEW

**Comment by Ismailtlem** [comment]
Any feedback on this ?

File: src/css/App.css [hunk 1]
```diff
@@ -302,6 +304,15 @@ dd {
   #playContainer::before {
     display: none;
   }
+  .animation {
+    background-image: url('https://cdn-media-1.freecodecamp.org/code-radio/bg.jpg');
```
Comment (ShaunSHamilton) [inline src/css/App.css:308]: This is a large image. Is there not a smaller version we could use @ahmadabdolsaheb ?

Comment (ahmadabdolsaheb) [reply src/css/App.css:308]: We could load different images for different screen sizes, but we are only loading the background image for large screens any way.

Comment (ShaunSHamilton) [reply src/css/App.css:308]: Right, but with this line, we are loading a 1920x1013 image for screens less than 750px wide

Comment (Ismailtlem) [reply src/css/App.css:308]: Maybe load a blue screen for screens less than 750 px wide ? Like what we do for screens less than 500 px ?

**Comment by Ismailtlem** [comment]
Hi @ahmadabdolsaheb 
Why was this PR closed ? Is there something to fix ?

**Comment by ShaunSHamilton** [comment]
@Ismailtlem I believe this was closed, in response to renaming the default/main branch from `master` to `main`.

It will be necessary for you to reset your fork to follow the new branch and HEAD. Then, open a new PR.

**Pull request closed without merging**